.gitignore

```diff
@@ -11,4 +11,5 @@ Desktop.ini
 .DS_Store
 
 # NPM files
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+node_modules/
```

Comment:
Could you revert this change?  We'll probably want to do this eventually, but it doesn't make sense as part of this patch.